src/server/mailgun.ts

```diff
@@ -43,10 +58,201 @@ export const sendWelcomeEmail = async (emailTo: string, fullName: string) => {
             template: 'welcome email',
             'h:X-Mailgun-Variables': JSON.stringify({
                 fullName,
-                resetPasswordLink: `https://${process.env.DOMAIN_NAME}/account/reset-password`,
+                resetPasswordLink: `${BASE_URL}/account/reset-password`,
             }),
         })
     } catch (error) {
-        logger.error.log('Welcome email error: ', error)
+        logger.error.log('sendWelcomeEmail error: ', error)
+    }
+}
+
+export const sendStudyProposalEmails = async (studyId: string) => {
+    const [mg, domain] = await mailGunConfig()
+    if (!mg) return
+
+    const { study, org } = await getStudyAndOrg(studyId)
+    const reviewersToNotify = await getUsersByRoleAndOrgId('reviewer', org.id)
+
+    for (const reviewer of reviewersToNotify) {
+        const email = reviewer.email
+        if (!email) continue
+
+        try {
+            await mg.messages.create(domain, {
+                from: SI_EMAIL,
+                to: email,
+                subject: 'SafeInsights - New study proposal',
+                template: 'vb - new research proposal',
+                'h:X-Mailgun-Variables': JSON.stringify({
+                    fullName: reviewer.fullName,
+                    studyTitle: study.title,
+                    submittedBy: study.researcherName,
+                    submittedOn: dayjs(study.createdAt).format('MM/DD/YYYY'),
+                    studyURL: `${BASE_URL}/organization/${org?.slug}/study/${studyId}/review`,
+                }),
+            })
+        } catch (error) {
+            logger.error.log('sendStudyProposalEmails error: ', error)
+        }
+    }
+}
+
+export const sendStudyProposalApprovedEmail = async (studyId: string) => {
+    const [mg, domain] = await mailGunConfig()
+    if (!mg) return
+
+    const { study, org } = await getStudyAndOrg(studyId)
+
+    const researchersToNotify = await getUsersByRoleAndOrgId('researcher', org.id)
```

Comment:
I think we're only supposed to notify the researcher who created the study right? not every researcher in the org

Reply:
Checking with @iccgoncalves on this, we talked a while ago about the recipient rules and [the ticket](https://openstax.atlassian.net/browse/OTTER-100) says:

```Reviewer == all Reviewers in that organization

Researcher == all Researchers in that organization```